jest.setup.js

```diff
@@ -1,9 +1,3 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
-// allows you to do things like:
-// expect(element).toHaveTextContent(/react/i)
-// learn more: https://github.com/testing-library/jest-dom
-import "@testing-library/jest-dom";
```

Comment:
This was not needed (loaded automatically) - and was causing errors due to Jest dependency not being available during setup. (and therefore, no `expect`)